internal/cloudapi/v2/server.go

```diff
@@ -87,6 +89,12 @@ func (s *Server) Handler(path string) http.Handler {
 	e.Use(middleware.Recover())
 	e.Logger = common.Logger()
 
+	if sentry.CurrentHub().Client() == nil {
+		logrus.Warn("Sentry/Glitchtip not initialized, echo middleware was not enabled")
```

Comment:
I think I would apply a similar suggestion to my previous comment.

Reply:
I agree about the other one, but this one makes sense since it checks if sentry itself, but I agree it can be improved, may be "Sentry/Glitchtip not initialized, skipping echo middleware".

Reply:
Right, makes sense, I misread the code initially.